library/alpine

```diff
@@ -4,119 +4,140 @@
-amd64-GitCommit: f57570e8dafadf7ab0116548bfdcc6a9b7ffc25a
+amd64-GitCommit: e2356fa734792d004f8dde5e515b42b6f350a45f
 amd64-Directory: versions/library-3.6/x86_64
 
 Tags: 3.7
 Architectures: arm64v8, arm32v6, ppc64le, s390x, i386, amd64
 arm64v8-GitFetch: refs/heads/rootfs/library-3.7/aarch64
-arm64v8-GitCommit: ff0663f124e074c4ac5047b4b8aae5866b2744b1
+arm64v8-GitCommit: 25892bedf763eaad549063e58a94dc498b4dae65
 arm64v8-Directory: versions/library-3.7/aarch64
 arm32v6-GitFetch: refs/heads/rootfs/library-3.7/armhf
-arm32v6-GitCommit: bc9679c14d98f6554546da9d216bbe1818cd63d4
+arm32v6-GitCommit: af9121fc400c1bca157b4e0b55c75bd243e74915
 arm32v6-Directory: versions/library-3.7/armhf
 ppc64le-GitFetch: refs/heads/rootfs/library-3.7/ppc64le
-ppc64le-GitCommit: 36d6cc31cc7a5bb901081ef509dc32f21893909e
+ppc64le-GitCommit: 87eaee82e4c3a25db0534190907ee91513187d53
 ppc64le-Directory: versions/library-3.7/ppc64le
 s390x-GitFetch: refs/heads/rootfs/library-3.7/s390x
-s390x-GitCommit: 32992d93a7c6e2f20febe09ada3edffc784cd2ce
+s390x-GitCommit: af2a01ed58e51d3eb8afbc93f9be9bf2e91f75cd
 s390x-Directory: versions/library-3.7/s390x
 i386-GitFetch: refs/heads/rootfs/library-3.7/x86
-i386-GitCommit: 6d29ed90a630b34bb0e35f04bbed1ff1dee47b35
+i386-GitCommit: 677a0ed1eacb5214836a864afded2e366abe41fe
 i386-Directory: versions/library-3.7/x86
 amd64-GitFetch: refs/heads/rootfs/library-3.7/x86_64
-amd64-GitCommit: 04bf61530405ca0b83b5dc887d0b9d0718c151fe
+amd64-GitCommit: ca1eb9308380da0be8042c9af7a4bbe5ec00f34b
 amd64-Directory: versions/library-3.7/x86_64
 
 Tags: 3.8, latest
 Architectures: arm64v8, arm32v6, ppc64le, s390x, i386, amd64
 arm64v8-GitFetch: refs/heads/rootfs/library-3.8/aarch64
-arm64v8-GitCommit: 38436e83873bbeea3935947029d44ae894a1cc5a
+arm64v8-GitCommit: 21fbb807e724fb80d5c1ab1577c89d8ddfbfe08e
 arm64v8-Directory: versions/library-3.8/aarch64
 arm32v6-GitFetch: refs/heads/rootfs/library-3.8/armhf
-arm32v6-GitCommit: bea52cd0742bd7cd01477472938d619f58b77327
+arm32v6-GitCommit: a274ddc4574ec0ae5be1b386a7d4312c7e499ca9
 arm32v6-Directory: versions/library-3.8/armhf
 ppc64le-GitFetch: refs/heads/rootfs/library-3.8/ppc64le
-ppc64le-GitCommit: 9d3a00da1d2f0daad611f96a9dc967fc68cc5b88
+ppc64le-GitCommit: 6eaaf1e213b53bb20e596f3ef8cb848878974b11
 ppc64le-Directory: versions/library-3.8/ppc64le
 s390x-GitFetch: refs/heads/rootfs/library-3.8/s390x
-s390x-GitCommit: 9fa40ec5fe4be402751c441e6e89eef390e29427
+s390x-GitCommit: 5d9396de964f3accc5971521d5b684588c859990
 s390x-Directory: versions/library-3.8/s390x
 i386-GitFetch: refs/heads/rootfs/library-3.8/x86
-i386-GitCommit: e8dbc6e8cafd058246d3dd686156f30ec8a5ce89
+i386-GitCommit: 197f1e39f11e16d1eb06dfed7452b7b1e3674a9e
 i386-Directory: versions/library-3.8/x86
 amd64-GitFetch: refs/heads/rootfs/library-3.8/x86_64
-amd64-GitCommit: d39bc7743955e37ac15e8d82af14410c0b8b9a03
+amd64-GitCommit: 63a4b585539aec349a3b1b5109be7bfece856cff
 amd64-Directory: versions/library-3.8/x86_64
 
+Tags: 3.9
```

Comment:
Did you want to move `latest` too since 3.9 is GA now?

Reply:
Yes, but I will have to do that later. I dont have time today.

Reply:
No worries, I'll push a commit to update it here and we can fix the generate script separately. :+1: